base/server/upgrade/11.9.0/01-EnableEST.py

```diff
@@ -135,6 +153,43 @@ def update_registry(self, subsystem):
                 'RA Client Authentication Information Input'
             subsystem.save()
 
+    def register_ca_auth_for_est(self, subsystem):
+        """Register CMCAuthForEST authentication module in CS.cfg"""
+        self.backup(subsystem.cs_conf)
+
+        # Check if CMCAuthForEST implementation is already registered
+        auth_impl_ids = subsystem.config.get('auths.impl._000', '')
```

Comment:
IIUC this code is getting the value of a comment param. You probably want to get the value of `auths.impl.CMCAuthForEST.class` instead.